modules/microshift-olm-deploy-ops-spec-ns.adoc

```diff
@@ -133,7 +133,7 @@ spec:
 ----
 $ oc apply -f _<my-catalog-source.yaml>_ <1>
 ----
-<1> Replace _<my-catalog-source.yaml>_ with your catalog source configuration file name.
+<1> Replace `<my-catalog-source.yaml>` with your catalog source configuration file name.
```

Comment:
```suggestion
<1> Replace `<my_catalog_source>.yaml` with your catalog source configuration file name.
```

Reply:
chnages are made in both lines 134 and 136. Is that fine?